common-npm-packages/codecoverage-tools/Tests/data/fakeData.ts

```diff
@@ -0,0 +1,31 @@
+import * as cheerio from 'cheerio';
```

Comment:
Is it used somewhere?

Reply:
Removed it for now. It will be used in next suites